pkgs/development/python-modules/devito/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, anytree
+, nbval
+, flake8
+, sympy
+, scipy
+, cached-property
+, psutil
+, py-cpuinfo
+, cgen
+, click
+, multidict
+, distributed
+, pyrevolve
+, codepy
+, pytest
+, pytest-xdist
+, matplotlib
+, mpi
+}:
+
+buildPythonPackage rec {
+  pname = "devito";
+  version = "4.6.2";
+
+  src = fetchFromGitHub {
+    owner = "devitocodes";
+    repo = "devito";
+    rev = "v${version}";
+    sha256 = "sha256-1yPKaVtxjeoANCUAqJYZndi9C/1BVVg5vuNNAb19zvs=";
+  };
+
+  patches = [ ./clear-deps.patch ];
```

Comment:
We want to do that with substituteInPlace in postPatch. Patches often need to be rebased which causes more maintenance burden.

Reply:
I see, ok then!